content/learn/book/your-first-game/_index.md

```diff
@@ -0,0 +1,685 @@
+
+```rs
+.insert_resource(ClearColor(Color::rgb(0.9, 0.9, 0.9)))
+``` 
+Here we insert a ClearColor resource, set to a light grey. (rgb stands for the red, green and blue parts of the final color.)
+
+```rs
+.add_startup_system(setup.system())
+``` 
+Here we register our very own setup function as a startup system.
+Startup systems are only called once and before any of the normal systems are called.
+We will cover the setup function code in detail in a moment.
+
+```rs
+.add_system(paddle_movement_system.system())
+``` 
+Here we add our very own paddle_movement_system function as a system.
+Systems are called every frame.
+We will cover the paddle_movement_system function code in detail in a moment.
+
+```rs
+.run();
+``` 
+Finally we tell the Bevy game engine to start with a call to run.  
+(If you are really interested in the source code of the run function you can find it here: [app.rs](https://github.com/bevyengine/bevy/blob/latest/crates/bevy_app/src/app.rs))
+
+## Let's take a look at that setup function.
+
+```rs
+struct Paddle {
+    speed: f32,
+}
+
+enum Collider {
+    Solid,
+    Scorable,
+    Paddle,
+}
+
+fn setup(
+    commands: &mut Commands,
+    mut materials: ResMut<Assets<ColorMaterial>>,
+    asset_server: Res<AssetServer>,
+) {
+    commands
+        // cameras
+        .spawn(OrthographicCameraBundle::new_2d())
+        .spawn(UiCameraBundle::default())
+        // paddle
+        .spawn(SpriteBundle {
+            material: materials.add(Color::rgb(0.5, 0.5, 1.0).into()),
+            transform: Transform::from_xyz(0.0, -215.0, 0.0),
+            sprite: Sprite::new(Vec2::new(120.0, 30.0)),
+            ..Default::default()
+        })
+        .with(Paddle { speed: 500.0 })
+        .with(Collider::Paddle);
+}
+``` 
+`struct Paddle` is a struct with one float field called speed.  
+`enum Collider` is an enumeration with three fields. We will use it to declare the type of collisions the ball can hit.
+
+Next our setup function expects three arguments:  
+ * a reference to a [Commands](https://docs.rs/bevy_ecs/latest/bevy_ecs/struct.Commands.html) struct.
+ * an instance of a collection of ColorMaterials.
+ * an instance of AssetServer.
+
+The `Commands` struct can be used to spawn `Bundles` and to add `Components`.  
+Material is used to paint 2D or 3D objects on screen and `ColorMaterial` is the simplest Material as it only supports a single color as oposed to, for example, textures.  
+The AssetServer will be used in the last section of this tutorial to load the font we will use to display the game score.
+
+Inside the function (that is only run once, since it is only a startup_system) we then use the commands struct
+to spawn (or in another word, add) a couple of Bundles. Namely we spawn an orthographic and a separate UI camera.
```

Comment:
```suggestion
to spawn (or in another word, add) a couple of Bundles,  spawning both an orthographic (top-down) and a separate UI camera.
```

Reply:
Like above, I'd prefer to avoid adding the UI camera until it actually has an effect.